packages/vscode-dependencies-validation/src/diagnostics/refreshDiagnostics.ts

```diff
@@ -0,0 +1,29 @@
+import type { DiagnosticCollection, Uri } from "vscode";
+import { readFile } from "fs-extra";
+import { findDependencyIssues } from "@sap-devx/npm-dependencies-validation";
+import { convertToDiagnostics } from "./convertToDiagnostics";
+
+// TODO: evaluate if we make a small unit test for this or not? (probably yes... as it does not seem hard)
```

Comment:
TODO: test?

Reply:
Wrote a test for this but it requires too much investment compare to the complexity of the code and the chance that this trivial 4 line function would cause a "real" bug...